### PR TITLE
dev container schema: use unevaluatedProperties

### PR DESCRIPTION
**Pull request by aeschli**

### CODE REVIEW

**Comment by aeschli** [comment]
See also https://github.com/devcontainers/spec/pull/167

File: extensions/configuration-editing/package.json [hunk 1]
```diff
@@ -140,11 +140,11 @@
       },
       {
         "fileMatch": "devcontainer.json",
-        "url": "./schemas/devContainer.schema.generated.json"
+        "url": "./schemas/devContainer.schema.json"
```
Comment (chrmarti) [inline extensions/configuration-editing/package.json:143]: Couldn't we reference the original like https://github.com/microsoft/vscode/blob/main/extensions/html-language-features/package.json#L253 does?

**Review by chrmarti — CHANGES_REQUESTED** [review]
Thanks! Left a question.

File: extensions/configuration-editing/package.json (hunk 1 above)
Comment (aeschli) [reply extensions/configuration-editing/package.json:143]: IMO it's better to avoid fetching the schema over the web so that all works offline

**Review by chrmarti — APPROVED** [review]
Thanks!

**Pull request merged**